kotlin-analysis-api/testData/annotationValue/annotationValue_java.kt

```diff
@@ -24,8 +24,6 @@
 // MyClassInLib: stringParam = 2
 // MyClassInLib: stringParam2 = 1
 // MyClassInLib: stringArrayParam = [3, 5, 7]
-// []
```

Comment:
Accidental removal?

Reply:
This is intended. The annotation class has two parameters that have no explicit default value declared.

Reply:
To be clear,
1. `KSAnnotation.arguments` doesn't include default arg if the corresponding parameter doesn't have a explicit default value.
2. `KSAnnotation.defaultArguments` returns the type default value (0 for numerics, null for others) if no explicit default is specified.

This makes missing arguments detectable in `KSAnnotation.arguments`. `KSAnnotation.defaultArguments` should probably align with that, but I'm worrying that changing it would break a lot of processors.